### PR TITLE
Modified logging to take into account non deployable packages that we…

### PR DESCRIPTION
**Pull request by jacksondelametter**
Logging was not completely accurate in terms of what was actually being deployed and not being deployed. Also updating some logs.

### CODE REVIEW

File: src/artifacts_poetry_plugin/ArtifactsDeployHelper.py [hunk 1]
```diff
@@ -133,18 +133,40 @@ def get_poetry(self, root, files):
             poetry_root = Path(root)
             poetry = Factory().create_poetry(poetry_root)
             poetry.root_dir = poetry_root
-            logger.info(f"Found python project at {root}")
+            logger.debug(f"Found python project at {root}")
         return poetry
 
-    def log_non_deployable_packages(self, poetry_list):
-        package_set = set()
+    def get_package_dict(self, packages):
```
Comment (aaron-gary) [inline src/artifacts_poetry_plugin/ArtifactsDeployHelper.py:139]: Recommend using dictionary comprehension for this
i.e. 
package_dict = {x.name: x for x in packages}

**Review by carter-cundiff — APPROVED** [review]

File: src/artifacts_poetry_plugin/ArtifactsDeployHelper.py [hunk 2]
```diff
@@ -133,18 +133,40 @@ def get_poetry(self, root, files):
...
+                filter_total_non_deployable_dict.pop(package_key)
+        return list(total_deployable_dict.values()), list(
+            filter_total_non_deployable_dict.values()
+        )
+
+    def log_non_deployable_packages(self, poetry_list):
+        _, total_non_deployable = self.filter_deployable_non_deployable(poetry_list)
+        for package in total_non_deployable:
+            logger.warn(
+                f"Could not find any wheel or tar.gz files for package {package.name}:{package.pretty_version}. Package may be statically linked or doesnt exist in artifacts cache"
```
Comment (aaron-gary) [inline src/artifacts_poetry_plugin/ArtifactsDeployHelper.py]: "doesn't" is missing the apostrophe

**Review by aaron-gary — APPROVED** [review]

File: src/artifacts_poetry_plugin/ArtifactsDeployHelper.py (hunk 1 above)
Comment (jacksondelametter) [reply src/artifacts_poetry_plugin/ArtifactsDeployHelper.py:139]: Fixed.

File: src/artifacts_poetry_plugin/ArtifactsDeployHelper.py (hunk 2 above)
Comment (jacksondelametter) [reply src/artifacts_poetry_plugin/ArtifactsDeployHelper.py]: Fixed.

**Pull request merged**